ibis/omniscidb/client.py

```diff
@@ -764,6 +764,36 @@ def _execute_query(self, dml, **kwargs):
         query = self.query_class(self, dml, **kwargs)
         return query.execute(**kwargs)
 
+    def execution_mode(self, processing_unit: str):
+        """
+        Define where the SQL execution occurs (GPU or CPU).
+
+        Use "execution_mode" before the expression execution to change
+        where it should be executed in the backend engine.
+
+        Parameters
+        ----------
+        processing_unit : {'cpu', 'gpu'}
+          Define if the request will be executed on CPU or GPU.
+
+        Examples
+        --------
+        >>> con.execution_mode('gpu') # doctest: +SKIP
+        >>> expr.execute()  # doctest: +SKIP
+        >>> con.execution_mode('cpu') # doctest: +SKIP
+        """
+        if processing_unit not in ['cpu', 'gpu']:
```

Comment:
```suggestion
        if processing_unit not in PROCESSING_UNIT_MAP:
```

If you define `PROCESSING_UNIT_MAP` before.